sample-root-scripts/MakeKin.py

```diff
@@ -45,7 +44,7 @@
                   help=" Average number of vertices to simulate per event. Default: %s" \
                   % (verticesPerEventDefault),
                   metavar="#", default=verticesPerEventDefault)
-optchoices = pid.keys()
+optchoices = list(pid.keys())
 optdefault = "mu-"
```

Comment:
```suggestion
optdefault = None
parser.add_option("-s", "--seed", dest="seed",
                  help="Random number seed to use. Default: None (use system time)",
                  metavar="SEED", default=optdefault)
optchoices = pid.keys()
optdefault = "mu-"
```

Reply:
And then immediately after parsing the arguments, need (you didn't edit a line close enough so github wouldn't let me suggest for this one)
```
random.seed(options.seed)
```